src/main/cli/cli.c

```diff
@@ -4829,6 +4829,10 @@ if (buildKey) {
     cliPrintLinefeed();
 #endif // USE_GPS
 
+#ifdef LOCALE
```

Comment:
I am not sure I understand. If not defined then not included in code, I think like USE_GPS USE_OSD etc
Or do you simply want me to remove the #ifdef xx #endif part, since LOCALE always are defined ?

Reply:
There is no fallback without defining locale (and LOCALE).